common/reviews/api/node-library-build.api.ts

```diff
@@ -1,13 +1,9 @@
 export declare function addSuppression(str: string): void;
 
 class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
-  // (undocumented)
```

Comment:
@cliffkoh We're going to push a fix to your branch in a few mins, then you should be good to merge.